api/pendingactions/pendingactions.go

```
@@ -56,57 +56,71 @@ func (service *PendingActionsService) Create(r portainer.PendingActions) error {
 }
 
 func (service *PendingActionsService) Execute(id portainer.EndpointID) error {
```

This doesn't need to return an error anymore